components/rmrk/Gallery/History.vue

```diff
@@ -298,7 +299,7 @@ export default class History extends mixins(ChainMixin, KeyboardEventsMixin) {
       event['Block'] = String(newEvent['blockNumber'])
 
       // ID for b-table: Use a unique key of your data Object for each row.
-      event['ID'] = newEvent['id']
+      event['ID'] = newEvent['timestamp']
```

Comment:
unique key of  for each row may be undefined
list event: 
<img width="406" alt="image" src="https://user-images.githubusercontent.com/31397967/159828731-f502491a-9e7a-4712-9d09-8cab972cf1de.png">

Reply:
You can return id for the events :)

Reply:
This line of code does not affect the display of the  'from'、'to' and 'amount' in most of time.
Can we open another issue to handle this ‘id’?  

id has been requested but not returned @vikiival 
<img width="426" alt="image" src="https://user-images.githubusercontent.com/31397967/159903447-604f2978-6605-49d8-8c66-bb29dac608a5.png">

Reply:
If you don't think it's appropriate, I can delete this line of code.

Let major bugs be fixed first

Reply:
> id has been requested but not returned

hmm that's super strange

Reply:
I moved this to #2589